docs/faq/index.rst

```diff
@@ -1,17 +1,276 @@
+* `Demo: High Availability with Marathon and Flocker <https://clusterhq.com/2015/10/06/marathon-ha-demo/>`_
+* `Github: Using Mesos isolator for Flocker <https://github.com/ClusterHQ/mesos-module-flocker>`_
+
+Can I run Flocker with local storage?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Flocker integrates with many storage platforms including software defined storage platforms like EMC ScaleIO, Hedvig, Ceph, and ConvergIO.
+
+Flocker is not a platform that manages local storage such as existing HDD or SSD disks on your server, so you should choose a backend that is suitable for your needs.
+
+For more information, see :reF:`storage-backends`.
+
+Can I contribute a new Flocker storage backend?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Yes, for more information see :ref:`contribute-flocker-driver`.
 
 There are additional FAQs specifically relating to contributing a new Flocker storage backend.
 These can be found :ref:`here <build-flocker-driver-faq>`.
+
+Can more than one container access the same volume?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Flocker works by creating a 1 to 1 relationship of a volume and a container.
+This means you can have multiple volumes for one container, and those volumes will always follow that container.
+
+Flocker attaches volumes to the individual agent host (docker host) and this can only be one host at a time because Flocker attaches Block-based storage.
+Nodes on different hosts cannot access the same volume, because it can only be attached to one node at a time.
+
+If multiple containers on the same host want to use the same volume, they can, but be careful because multiple containers accessing the same block storage volume can cause corruption.
+
+In order for Flocker to support multiple attachments it would need to support a network filesystem like NFS, GlusterFS or multi-attach based storage.
+
+Can Flocker work across availability zones in AWS or regions in Cinder?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Not currently, this is because volumes are only allowed to attach to nodes in their zone or region.
+
+We hope to eventually work on support to allow your volumes to move from one zone or region to another.
+
+Can Flocker work with multiple :file:`agent.yml` configurations?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Not currently, a single Flocker cluster can only be configured with one storage backend at a time.
+
+What other storage backends does it work with?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Please view our supported storage backends here: :ref:`storage-backends`
+
+Is there performance benchmarks vs NFS?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+No, we are attacking mainly block storage use cases where volumes are attached via iSCSI or Fiber Channel.
+You can use NFS and block storage side by side, they are not exclusive.
+
+Can I attach a single volume to multiple hosts?
```

Comment:
I would move this next to "Can more than one container access the same volume" as it is related.